openlibrary/plugins/ol_infobase.py

```diff
@@ -480,7 +480,10 @@ def process_edition_doc(self, doc):
 
     def normalize_edition_title(self, title):
         if isinstance(title, str):
```

Comment:
```suggestion
        if isinstance(title, bytes):
```
On Python 2, `bytes == str` so this changes nothing.
On Python 3, this makes the following change unnecessary so you can revert it.